luarules/gadgets/api_ranks.lua

```diff
@@ -1,16 +1,13 @@
--- $Id: api_ranks.lua 3171 2008-11-06 09:06:29Z det $
---------------------------------------------------------------------------------
---------------------------------------------------------------------------------
 
 function gadget:GetInfo()
 	return {
-		name = "ranks api",
-		desc = "a shared interface to recieve unit ranks",
-		author = "jK",
-		date = "Dec 19, 2007",
+		name    = "ranks api",
+		desc    = "a shared interface to recieve unit ranks",
+		author  = "jK",
+		date    = "Dec 19, 2007",
```

Comment:
I don't see a mention of these whitespace changes in the PR description. Are they intentional? (Personally I prefer single spaces, unaligned, like it was without this change)

Reply:
Nah, just fixing a minor annoyance as I went through them. I'm happy to undo that if others don't agree.

Reply:
I prefer the single space each side of the equals sign, makes it easier to search for

Reply:
The people have spoken, all whitespace changes reverted 
![image](https://github.com/beyond-all-reason/Beyond-All-Reason/assets/3493638/a1c7e2d5-4115-4ad4-a9f7-00ec71be92a9)